gateway_code/open_nodes/node_rtl_sdr.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding:utf-8 -*-
+
+# This file is a part of IoT-LAB gateway_code
+# Copyright (C) 2015 INRIA (Contact: admin@iot-lab.info)
+# Contributor(s) : see AUTHORS file
+#
+# This software is governed by the CeCILL license under French law
+# and abiding by the rules of distribution of free software.  You can  use,
+# modify and/ or redistribute the software under the terms of the CeCILL
+# license as circulated by CEA, CNRS and INRIA at the following URL
+# http://www.cecill.info.
+#
+# As a counterpart to the access to the source code and  rights to copy,
+# modify and redistribute granted by the license, users are provided only
+# with a limited warranty  and the software's author,  the holder of the
+# economic rights,  and the successive licensors  have only  limited
+# liability.
+#
+# The fact that you are presently reading this means that you have had
+# knowledge of the CeCILL license and that you accept its terms.
+
+""" Open Node Rtl SDR experiment implementation """
+
+import logging
+import shlex
+
+from gateway_code.common import logger_call
+from gateway_code.utils import subprocess_timeout
+from gateway_code.utils.rtl_tcp import RtlTcp
+from gateway_code.open_nodes.common.node_no import NodeNoBase
+
+LOGGER = logging.getLogger('gateway_code')
+
+# This command controls the power of the open node/rtl_tcp USB stick via the
+# Yepkit module.
+YKUSHCMD = "sudo ykushcmd {model} {cmd} {port}"
+
+
+class NodeRtlSdr(NodeNoBase):
+    """ Open node Rtl SDR implementation """
+
+    TYPE = 'rtl_sdr'
+    RTL_TCP_PORT = 50000
+    RTL_TCP_FREQ = 868000000
+
+    def __init__(self):
+        self.rtl_tcp = RtlTcp(self.RTL_TCP_PORT, self.RTL_TCP_FREQ)
+
+    @staticmethod
+    def _call_cmd(command_str):
```

Comment:
this method is really starting to be repeated every where, don't you think ?

Reply:
I agree. And I think I'll need it again very soon but #89 won't be adapted for that. There's no need for a base class just for that. I'll open a PR (an alternative to #89)

Reply:
see #98

Reply:
Now that I'm thinking at this a second time, I think reusing `subprocess_timeout` is useless in this case. Just a regular `subprocess.call` will be enough, in try/except to catch errors. The underlying command is not supposed to take time. I'll change that.